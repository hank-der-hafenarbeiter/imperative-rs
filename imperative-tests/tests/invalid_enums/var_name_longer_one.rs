use imperative_rs::InstructionSet;
#[derive(InstructionSet)]
enum Inst {
    #[ opcode = "0xff_vw_ff" ]
    A{vw:u8},
}

fn main() {}