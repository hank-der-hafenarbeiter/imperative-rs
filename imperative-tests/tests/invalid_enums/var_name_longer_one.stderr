error: Variables names must be one symbol long
 --> $DIR/var_name_longer_one.rs:5:7
  |
5 |     A{vw:u8},
  |       ^^

error: Instruction contains invalid variable names. See above.
 --> $DIR/var_name_longer_one.rs:5:7
  |
5 |     A{vw:u8},
  |       ^^
